css/src/components/messages.scss

```diff
@@ -0,0 +1,58 @@
+.messages {
```

Comment:
Please re-read the components guide here, we're missing a few conventions in this PR: 
https://design.learn.microsoft.com/components/overview.html

Reply:
Especially naming conventions, and creating separate components for separate concerns

Reply:
Also, please revisit contributor checklist in PR description

Reply:
let me know if the naming is better